config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow_devmodeWithExternalResource.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: sw.kogito.kie.org/v1alpha08
```

Comment:
not sure whether this sample is useful. This `KogitoServerlessWorkflow` is already defined in `sw.kogito_v1alpha08_kogitoserverlessworkflow_devmodeWithConfigMapAndExternalResource.yaml` file and that latter one is clearer IMO
no need to replicate half of the information in another file
@MarianMacik @davidesalerno wdyt ?

Reply:
@radtriste This is a complete example configmap + workflow requested by Felipe here https://github.com/kiegroup/kogito-serverless-operator/pull/81#discussion_r1137525179

Reply:
ok but the "old" one, aka `sw.kogito_v1alpha08_kogitoserverlessworkflow_devmodeWithExternalResource` is useless IMO as non fully working

Reply:
created https://issues.redhat.com/browse/KOGITO-8961 to address this